ImperatorToCK3/Source/CK3/Titles/LandedTitles.h

```diff
@@ -25,6 +25,8 @@ class LandedTitles: commonItems::parser
 
 	[[nodiscard]] std::optional<std::string> getCountyForProvince(int provinceID);
 
+	std::optional<int> capitalBarony;	// Capital barony (for counties)
```

Comment:
there is no effective difference between optional<int> and int=0. Both will do the same boolean check if(capitalBarony).

Reply:
✔️